src/utils/FunctionSelectorDecoder.sol

```diff
@@ -104,6 +104,17 @@ library FunctionSelectorDecoder {
                 require(customError.length - innerErrorOffset >= innerErrorLength, "Invalid Error(string)");
                 reason = string(_decodeInnerError(customError, innerErrorOffset));
             }
+
+            /////////////////////////////
+            /// Eigenlayer Custom Errors
+            /////////////////////////////
+
+            // StrategyNotWhitelisted()
+            if (innerErrorSelector == 0x5dfb2ca2) {
+                reason = string("StrategyNotWhitelisted()");
```

Comment:
Need to add more errorSelectors from Eigenlayer, they moved from require() statements.
This is mainly for bubbling-up user friendly errors.